cmd/ctl/pkg/status/certificate/certificate.go

```diff
@@ -150,21 +151,88 @@ func (o *Options) Run(args []string) error {
 	fmt.Fprintf(o.Out, fmt.Sprintf("Not After: %s\n", formatTimeString(crt.Status.NotAfter)))
 	fmt.Fprintf(o.Out, fmt.Sprintf("Renewal Time: %s\n", formatTimeString(crt.Status.RenewalTime)))
 
+	// TODO: What about timing issues? When I query condition it's not ready yet, but then looking for cr it's finished and deleted
```

Comment:
That is a very good point. I don't know that k8s API allows for getting a consistent snapshot.

It's possible that we just have to live with giving confusing output at times, but it's good to keep it
in mind while writing this code that just because the CR is in one state doesn't mean that the Cert
has to be in the corresponding state.